canvas_modules/common-canvas/src/common-properties/components/title-editor/title-editor.jsx

```diff
@@ -137,19 +137,67 @@ class TitleEditor extends Component {
 				hasIconOnly
 			/>);
 
-		const helpButton = this.props.help
-			? (<Button
-				kind="ghost"
-				className="properties-title-editor-btn help"
-				data-id="help"
-				onClick={this.helpClickHandler}
-				tooltipPosition="bottom"
-				renderIcon={Help}
-				size="sm"
-				iconDescription={helpButtonLabel}
-				hasIconOnly
-			/>)
-			: null;
+		const renderTooltip = (isDescWithLink) => {
+			const { description } = this.props;
+			// If description is present and has a link, show help button
+			const tooltipButton = isDescWithLink ? (
+				<ToggletipActions>
+					<Link
+						className="properties-title-editor-desc-btn desc-help"
+						onClick={this.helpClickHandler}
+						data-id="desc-help"
+					>
+						{helpButtonLabel}
+					</Link>
+				</ToggletipActions>
+			) : null;
+
+			return (
+				<span className="properties-title-desc-icon">
```

Comment:
span seems odd here.  Should this be a div instead?

Reply:
Removed span and updated styling.